tests/unit/deferred.py

```diff
@@ -34,11 +35,19 @@
 @pytest.fixture
 def configure():
     config_testsuite()
-    config_merge({
+
+    additional_config = {
         'extensions': {
             'hil.ext.obm.mock': ''
-        },
-    })
+            }
+        }
+
+    # if we are using sqlite's in memory db, then change uri to a db on disk
+    uri = config.cfg.get('database', 'uri')
+    if uri == 'sqlite:///:memory:':
+        additional_config['database'] = {'uri': 'sqlite:////tmp/hil.db'}
```

Comment:
Probably better to use the `tempfile` module, and delete the file when the test exits.

I'm surprised this is passing on sqlite -- we're running the tests in parallel, so I would expect tests to trample on each other using the same db. Can you run travis with an assert False in here, just to verify that it's actually going into the if statement?